docs/sources/mimir/configure/configure-blocking-queries.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Configure queries blocking
+description: Use the queries blocking feature
+weight: 100
+---
+
+# Configure queries blocking
+
+In certain situations, you may not be able to control the queries being sent to your Mimir installation. These queries
+may be intentionally or unintentionally expensive to run, and they may affect the overall stability or cost of running
+your service.
+
+You can block queries using [per-tenant overrides]({{< relref "./about-runtime-configuration" >}}), like so:
+
+```yaml
+overrides:
+  "tenant-id":
+    blocked_queries:
+      # block this query exactly
+      - pattern: 'sum(rate(node_cpu_seconds_total{env="prod"}[1m]))'
+
+      # block any query matching this regex pattern
+      - pattern: '.*env="prod".*'
+        regex: true
+```
+
+NOTE: To set up runtime overrides, refer to [runtime configuration]({{< relref "./about-runtime-configuration" >}}).
+
+**Note:** the order of patterns is preserved, so the first matching pattern will be used
```

Comment:
Notes follow the syntax at https://grafana.com/docs/writers-toolkit/write/shortcodes/#example.